src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -54,23 +63,19 @@
 
   @Override
   public InputSplit[] getSplits(JobConf job, int numSplits) throws IOException {
-    String tableDir = StringUtils.substringAfter(job.get("location"), "file:");
-
     //Change this to use whichever Catalog the table was made with i.e. HiveCatalog instead etc.
     HadoopTables tables = new HadoopTables(job);
+    String tableDir = StringUtils.substringAfter(job.get("location"), "file:");
```

Comment:
This is OK for now but what will happen if the path starts with s3:// ? We should add a test later  using one of our methods for "faking" s3 to test this.

Reply:
yeah definitely - i tried running a test on emr to see what would happen with the path changing, but i couldn't do anything without a serde working (otherwise everything fails) so i could pick up the investigation on this again now i think